ex1/ex1.c

```diff
@@ -9,6 +9,9 @@
 int main(void)
 {
     // Your code here
+    int x = 100;
+    fork();
```

Comment:
In this `ex` you want both the child and the parent to modify the value of x then print x.